bitnami/kubeapps/templates/NOTES.txt

```diff
@@ -35,7 +35,7 @@ To access Kubeapps from outside your K8s cluster, follow the steps below:
  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
        Watch the status by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "kubeapps.fullname" . }}'
 
-   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "kubeapps.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "kubeapps.fullname" . }} --template "{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}")
```

Comment:
Can you actually make this change upstream in https://github.com/kubeapps/kubeapps, we have a sync setup to auto update it here from there.

Reply:
Yes, of course